cdk/database/__init__.py

```diff
@@ -71,7 +71,8 @@ def dynamodb_old_table(self):
                                                  partition_key=aws_dynamodb.Attribute(
                                                      name='PK',
                                                      type=aws_dynamodb.AttributeType.STRING),
-                                                 billing_mode=aws_dynamodb.BillingMode.PAY_PER_REQUEST)
+                                                 billing_mode=aws_dynamodb.BillingMode.PAY_PER_REQUEST,
+                                                 time_to_live_attribute="ttl_expiration")
```

Comment:
probably makes more sense to call this something like "last_updated" but not a huge deal